x/merkledb/trie_test.go

```diff
@@ -43,7 +43,7 @@ func getNodeValue(t ReadOnlyTrie, key string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	if result.key != path || result == nil {
+	if result == nil || result.key != path {
```

Comment:
Similar comment - I think we could pull this into a small fix PR

Reply:
https://github.com/ava-labs/avalanchego/pull/2268